vulture.py

```diff
@@ -165,8 +167,14 @@ def _get_modules(self, paths, toplevel=True):
                 sys.exit('Error: %s could not be found.' % path)
         return modules
 
+    def _get_stdlib_whitelist_file(self):
+        script = os.path.abspath(stdlib.__file__)
```

Comment:
Even if we get the location of the `vulture.py`, can we have a relative path to `stdlib`?